### PR TITLE
Switch all encodings to camelCase

### PR DESCRIPTION
**Pull request by paf31**

### CODE REVIEW

File: ndc-reference/tests/query/table_argument_order_by/expected.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   {
     "rows": [
       {
-        "id": 2
+        "id": 1
       },
       {
-        "id": 1
+        "id": 2
```
Comment (sordina) [inline ndc-reference/tests/query/table_argument_order_by/expected.json:8]: Is this relevant to the casing change?

**Review by sordina — CHANGES_REQUESTED** [review]
@paf31 This looks good just two concerns:

* Can we be sure that we've covered all casing changes?
* Is this intended to be out for the October releases? What will the downstream adoption look like? Could we have an overview of what the subsequent changes downstream will be required in the PR description?

File: ndc-reference/tests/query/table_argument_order_by/expected.json (hunk 1 above)
Comment (paf31) [reply ndc-reference/tests/query/table_argument_order_by/expected.json:8]: No, good catch. I'll look into this, thanks.
